migrations/20230812123349_initial_db.down.sql

```diff
@@ -0,0 +1,6 @@
+-- Add down migration script here
```

Comment:
why use up & down scripts if I could just create a new db with one command?

Reply:
When a migration hasn't had the expected result, it is better to roll back and rewrite it, instead of rolling it back with a checkpoint etc.

Reply:
makes sense but I can just shred the entire db and initialize a new one with `docker run -p 5432:5432 --name postgres -e POSTGRES_PASSWORD=password -v ./db/init:/docker-entrypoint-initdb.d --rm postgres` where the volume mapping adds sql scripts that are being executed the first time the db starts

Reply:
Sure thing, but that's not something you could/should to in a production setting.